packages/windows/src/RPA/Windows/__init__.py

```diff
@@ -41,10 +41,10 @@ class Windows(WindowsElementsMixin, DynamicCore):
         channels:
```

Comment:
I would refactor from above this as well:

From
```
Ultimately this library will replace current library ``RPA.Desktop.Windows`` if seen as approriate.
```

To (or similar)
```
This library replaces the old deprecated ``RPA.Desktop.Windows``, so please use ``RPA.Windows`` instead.
```

Reply:
Change made